AccessibilityTests/Sources/AccessibilityTests.swift

```diff
@@ -78,7 +84,12 @@ final class AccessibilityTests: XCTestCase {
         }
     }
     
+    private static func isMatrixString(_ string: String) -> Bool {
+        MatrixEntityRegex.isMatrixRoomAlias(string) || MatrixEntityRegex.isMatrixUserIdentifier(string)
+    }
+    
     private static let partiallyUnsupportedDynamicTypeMessage = "Dynamic Type font sizes are partially unsupported"
+    private static let notHumanReadableMessage = ""
```

Comment:
I think we should be setting this and using it above?

```suggestion
    private static let notHumanReadableMessage = "Label not human-readable"
```